schedule/migration/migration_offline_media_x86_64_all.yaml

```diff
@@ -0,0 +1,38 @@
+name: migration_offline_x86_64.yaml
```

Comment:
It's better to update the name for 'all patterns',
Ex: schedule/migration/migration_offline_media_all_patterns_x86_64.yaml

Reply:
rename to `schedule/migration/migration_offline_media_all_patterns_x86_64.yaml`